### ISSUE TITLE
Autofocus on login username

### ISSUE THREAD

**Issue by brucestull** [brucestull/technology-and-perceptibility#83]
Upon login and signup page loads, the focus needs to be set to username field so User doesn't have to tab to or click on or use their preferred method to set focus on the username input field.

These are bugs on two separate views.

`<input type="text" autofocus>`

Reference:
[autofocus](https://www.w3schools.com/tags/att_input_autofocus.asp)

**Comment by brucestull** [comment]
Bug noted:
    - On login page load, the username block seems to have focus for a moment then the focus moves away.

**Comment by brucestull** [comment]
Covered by https://github.com/brucestull/technology-and-perceptibility/issues/174
